horiba_sdk/devices/single_devices/monochromator.py

```diff
@@ -155,6 +155,19 @@ async def home(self) -> None:
         """
         await super()._execute_command('mono_init', {'index': self._id})
 
+    async def is_homed(self) -> bool:
```

Comment:
rename to "is_initialized" as well as mono "mono.initalize()" instead mono.home()